### DISCUSSION TITLE
Build inconsistency?

### DISCUSSION THREAD

**Discussion by hadmut** [docker-library/rabbitmq#687] [General]
Hi, 

docker://docker.io/library/rabbitmq:alpine  seems to be from 
"Created": "2024-01-05T21:27:24Z",

while

docker://docker.io/library/rabbitmq:management-alpine seems to be from
 "Created": "2023-06-02T21:59:28Z",

Shouldn't they be built together and have the same age?

regards

**Comment by tianon** [comment]
This is because we're now building them with `SOURCE_DATE_EPOCH` set to the most-recent commit timestamp of the `Dockerfile` they're built from (see https://github.com/moby/buildkit/blob/v0.12.4/docs/build-repro.md#source_date_epoch).

**Comment by hadmut** [comment]
It doesn't look fresh this way:

% podman image pull docker.io/library/rabbitmq:management-alpine

% podman image list docker.io/library/rabbitmq:management-alpine
REPOSITORY                  TAG                IMAGE ID      CREATED       SIZE
docker.io/library/rabbitmq  management-alpine  a095b429d01b  7 months ago  180 MB


Usually, I do consider images as too old to use or abandoned if older than about 2 months after a pull.

**Comment by michaelklishin** [comment]
This should have been addressed by the fact that there were new releases and the images were rebuilt.

I don't think anyone wants to seriously discuss whether an image without updates in the last two months can be considered "abandoned".

Closing.
